server/lib/models/topic.go

```diff
@@ -10,6 +18,147 @@ type Topic struct {
 }
 
 func InitTopic() {
-	create_table(Topic{}, "topics", "Id")
+	t := create_table(Topic{}, "topics", "Id")
+	t.AddIndex("locale", "INDEX", []string{"Locale"})
+}
+
+const TOPIC_FETCH_LIMIT = 50
```

Comment:
__[gometalinter]__
* [golint] don't use ALL_CAPS in Go names; use CamelCase
* [golint] exported const TOPIC_FETCH_LIMIT should have comment or be unexported